docs/features/achievements.md

```diff
@@ -10,12 +10,7 @@ by contributors.
 
 ## Example
 
-The VauntDev [Example](https://github.com/VauntDev/example) repository provides an [example config](https://github.com/VauntDev/example/blob/main/.vaunt/config.yaml)
-that can be placed in any repository that has Vaunt installed to enable achievements.
-
-The only section that would need to be updated is the webhooks.
-You will need to change the repository to a repository in your installation if you want to setup webhooks for achievement notifications.
-Check out the [webhooks](webhooks.md) guide for information about setting up webhooks for your achievements.
+The VauntDev [Example](https://github.com/VauntDev/example) repository provides an example config that can be placed in any repository that has Vaunt installed to enable achievements.
```

Comment:
I don't think we want to link to the repo anymore 
- just show a bare bones achievement yaml in line - that uses img placeholders.

Reply:
Just updated this. Let me know if you were thinking something different.